tools/spectral/ipa/rulesets/IPA-105.yaml

```
@@ -29,3 +30,18 @@ rules:
     then:
       field: '@key'
       function: 'eachResourceHasListMethod'
+  xgen-IPA-105-list-method-response-is-get-method-response:
+    description: >-
+      The response body of the List method should consist of the same resource object returned by the Get method. Validation checks that the List method results items reference the same schema as the Get method response.<br/>
+      - Validation applies to List methods for resource collections only<br/>
+      - Validation applies to json response content only<br/>
```

No longer needed. You need to merge master if that were needed.
```suggestion
      - Validation applies to json response content only
```

---

Thanks!